assets/src/components/cd/cluster/ClusterMetadata.tsx

```diff
@@ -149,6 +152,11 @@ function MetadataCard({
           />
         </section>
       )}
+      {cluster.metadata && (
+        <Code language="json">
+          {formatJson(JSON.stringify(cluster.metadata))}
```

Comment:
why do you need to stringify -> parse -> stringify here?  My guess is the only issue you'll face is null checking, and the api should guarantee it's valid json